### DISCUSSION TITLE
What's different between init_bulk and init_surf when using a 2D material

### DISCUSSION THREAD

**Discussion by anatowinter** [deepmodeling/dpgen#954] [Q&A]
I want to init a 2D MoS2 nanorribon, but I don't know which kind of initset I should choose,if I want to calculate thermal conduction inplane, I don't think the system is a surface system.

**Comment by HuangJiameng** [comment]
`init_bulk` and `init_surf` is introduced in the [document](https://docs.deepmodeling.com/projects/dpgen/en/devel/init/init-bulk.html).  For what is done in `init_surf`, you can refer to #927.
